packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -594,6 +594,12 @@ function transform(codeGen: CodeGen): t.Expression {
             data.push(codeGen.genRef(ref));
         }
 
+        // Spread
```

Comment:
Follow existing comment format?
```suggestion
        // Properties: lwc:spread directive
```